src/libraries/System.Reflection.Metadata/src/System/Reflection/Metadata/BlobContentId.cs

```diff
@@ -94,7 +94,7 @@ public static unsafe BlobContentId FromHash(byte[] hashCode)
         }
 
         public bool Equals(BlobContentId other) => Guid == other.Guid && Stamp == other.Stamp;
-        public override bool Equals(object obj) => obj is BlobContentId && Equals((BlobContentId)obj);
+        public override bool Equals(object? obj) => obj is BlobContentId && Equals((BlobContentId)obj);
```

Comment:
NIT: We normally do small refactoring for such cases
`public override bool Equals(object? obj) => obj is BlobContentId blobObj && Equals(blobObj);`